### PR TITLE
[TVMC] Allow direct numpy inputs to run_module

### PR DESCRIPTION
**Pull request by CircleSpin**
When using TVMC from python it more convenient to pass numpy arrays as inputs rather than saved files. This PR moves the file reading into drive_run to maintain the existing behavior for command line TVMC while making the python API cleaner.

### CODE REVIEW

**Comment by CircleSpin** [comment]
@jwfromm @leandron @comaniac @mdw-octoml 

Thoughts?

File: python/tvm/driver/tvmc/runner.py [hunk 1]
```diff
@@ -230,8 +235,8 @@ def make_inputs_dict(inputs_file, shape_dict, dtype_dict, fill_mode):
 
     Parameters
     ----------
-    inputs_file : str
-        Path to a .npz file containing the inputs.
+    inputs : dict
```
Comment (comaniac) [inline python/tvm/driver/tvmc/runner.py:238]:
```suggestion
    inputs : dict, or None
```
I'm actually not quite sure how this type should be in numpy-style...in Python style I'll just write `Optional[Dict]`, but an argument with this type is usually `inputs=None`. Meanwhile, a function called "make_inputs_dict" but having `inputs=None` is also weird...

Another direction is keeping `inputs` to be `dict`, and it becomes user's resposibility to pass `{}` when no inputs. In this case, you don't need the None checker.

Comment (CircleSpin) [reply python/tvm/driver/tvmc/runner.py]: Hi Cody,
Thanks for the feedback. Just pushed an updated version as you suggested.

**Review by comaniac — APPROVED** [review]
LGTM

**Comment by jwfromm** [comment]
Thanks @CircleSpin and @comaniac. This is now merged.

**Pull request merged**